<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>autentication</class>
 <widget class="QMainWindow" name="autentication">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>443</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Autentication</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 12pt &quot;Poppins&quot;;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>380</y>
      <width>391</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="text_progressbar">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>381</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 8.5pt &quot;Poppins&quot;;</string>
    </property>
    <property name="text">
     <string>Autenticando Biometria</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="image">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>211</width>
      <height>301</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>biometria.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>410</y>
      <width>241</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton{
background-color: rgb(131, 198, 131);
color: white;
border-radius: 10px;
padding:5px;
font-size:18px;
font-weight: bold;
}

QPushButton#pushButton:hover{
	
	background-color: rgb(105, 198, 127);
}</string>
    </property>
    <property name="text">
     <string>Processar Autenticação</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>401</width>
      <height>271</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>443</width>
     <height>42</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
